README.md

```diff
@@ -30,6 +30,17 @@ requests need to be manually reviewed and merged by a human.
 It is ***highly recommended*** to also use [TagBot][tagbot], which automatically tags a release in your
 repository after the new release of your package is merged into the registry.
 
+### Requirements for packages in the General registry
+
+Packages registered in the General registry must be made in good faith.
+
+We do not accept packages or package registrations that:
+1. Are intentionally broken
+2. Are made in bad faith and/or with bad intentions
+3. Waste the time and/or energy of registry maintainers and/or members of the community
```

Comment:
The third point is not too clear.

Reply:
Again, quoting from Slack.

Reply:
I don't like #3. It's too subjective.

Reply:
The whole thing is subjective 🤷

Reply:
I'll take 3 out. Three people have disliked it.